src/AppAny.HotChocolate.FluentValidation.csproj

```
@@ -33,8 +33,8 @@
 <!--  </ItemGroup>-->
 
   <ItemGroup Label="Packages">
-    <PackageReference Include="FluentValidation" Version="11.9.0" />
-    <PackageReference Include="HotChocolate.Execution" Version="13.8.1" />
+    <PackageReference Include="FluentValidation" Version="11.10.0" />
+    <PackageReference Include="HotChocolate.Execution" Version="14.0.0" />
```

Consider changing the version to newst
```suggestion
    <PackageReference Include="HotChocolate.Execution" Version="14.1.0" />
```

---

Updated to 14.1.0